### PR TITLE
Drush 10 compatibility

### PR DESCRIPTION
**Pull request by dbosen**

### CODE REVIEW

**Comment by dbosen** [comment]
Failing and fixed tests can be observerved here:

Fail: https://travis-ci.com/github/BurdaMagazinOrg/thunder-web/builds/173801512

Fix: https://travis-ci.com/github/BurdaMagazinOrg/thunder-web/builds/173882635

File: src/Task/CloneSite.php [hunk 1]
```diff
@@ -134,9 +134,9 @@ protected function getDestinationSiteStatus() {
       $drushCmd->dir(DocrootResolver::getDocroot($this->getDestination()));
 
       $drushCmd
-        ->arg('status')
+        ->arg('core-status')
```
Comment (chrfritsch) [inline src/Task/CloneSite.php:137]: Sometimes you are replacing 'status' by 'core-status'. Sometimes not. Is that right?

File: composer.json [hunk 2]
```diff
@@ -23,11 +23,11 @@
   "prefer-stable": true,
   "license": "GPL-2.0-or-later",
   "require": {
-    "php": ">=5.6.0",
+    "php": ">7.3",
     "composer/composer": "^1.4",
     "consolidation/robo": "^1.0",
     "symfony/filesystem": "^3.2",
-    "drush/drush": "^8.1"
+    "drush/drush": "~10.0"
```
Comment (chrfritsch) [inline composer.json]: I am fine with only drush 10, or would it work with 9 also?

Comment (dbosen) [reply composer.json]: I think Drupal 9 does not have a purpose anymore, but sure all would/should work with 9 as well, we could go with 9||10

File: src/Task/CloneSite.php (hunk 1 above)
Comment (dbosen) [reply src/Task/CloneSite.php:137]: No, I just forgot to do that in the other places. Will use core-status in the other places as well, I prefer full commands to short-hand in code.

File: scripts/update-tester.php [hunk 3]
```diff
@@ -208,9 +208,9 @@ public function testUpdate(
     $drushCmd->dir($absoluteDestinationDocroot);
     $drushCmd->printOutput(FALSE);
     $drushCmd
-      ->arg('status')
+      ->arg('core-status')
       ->option('pipe')
-      ->option('show-passwords');
+      ->option('fields', 'db-password,drupal-version,uri,db-driver,db-hostname,db-port,db-username,db-name,db-status,bootstrap,theme,admin-theme,php-bin,php-conf,php-os,drush-script,drush-version,drush-temp,drush-conf,install-profile,root,site,files,private,temp', '=');
```
Comment (chrfritsch) [inline scripts/update-tester.php:213]: Could we use "drush core-status --fields=*" ?

Comment (dbosen) [reply scripts/update-tester.php:213]: I was actually thinking the other way around. Just use the fields, that are really needed. But I guess just providing the same fields as drush 8 does is fine for now. But I do not like to use "*" :-)

File: composer.json (hunk 2 above)
Comment (dbosen) [reply composer.json]: So, this works with Drupal 9 as well

https://travis-ci.com/github/BurdaMagazinOrg/thunder-web/builds/173992346#L1481

**Review by chrfritsch — APPROVED** [review]
Ok, well done 👍

**Pull request merged**